packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.routing.js

```diff
@@ -89,6 +90,15 @@ export default /* @ngInject */ ($stateProvider) => {
       isVersionSupported: /* @ngInject */ (clusterMinorVersion, versions) =>
         versions.includes(clusterMinorVersion.toString()),
 
+      isKubernetesInstalling: /* @ngInject */ (
+        $http,
+        projectId,
+        kubeId,
+      ) => () =>
+        $http
+          .get(`/cloud/project/${projectId}/kube/${kubeId}`)
+          .then(({ data }) => data.status === KUBE_INSTALLING_STATUS),
```

Comment:
1. To get rid on the extra function call, set this resolve as a boolean instead of a function
```suggestion
      isKubernetesInstalling: /* @ngInject */ (
        $http,
        projectId,
        kubeId,
      ) =>
        $http
          .get(`/cloud/project/${projectId}/kube/${kubeId}`)
          .then(({ data }) => data.status === KUBE_INSTALLING_STATUS),
```